osv2/bulk.go

```diff
@@ -26,6 +26,9 @@ type BulkRequest struct {
 
 	// Index determines the entire index for the request
 	Index string
+
+	// IgnoreResponseDocuments determines if the bulk response is parsed all the time or only when errors present in the response
+	IgnoreResponseDocuments bool
```

Comment:
This needs to be added to the `FromDomainBulkRequest` converter so the version agnostic value is copied over.